kuma/wiki/views/list.py

```diff
@@ -72,6 +76,7 @@ def tags(request):
     return render(request, 'wiki/list/tags.html', {'tags': tags})
 
 
+@block_user_agents
```

Comment:
Should we add https://developer.mozilla.org/en-US/docs/needs-review and https://developer.mozilla.org/en-US/docs/needs-review/* to robots.txt?